src/fireedge/src/modules/containers/Zones/index.js

```diff
@@ -0,0 +1,208 @@
+            useUpdateMutation={props.useUpdateMutation}
+            initialState={{
+              selectedRowIds: props.selectedRowsTable,
+            }}
+          />
+        )}
+        simpleGroupsTags={(props) => (
+          <GroupedTags
+            tags={props.selectedRows}
+            handleElement={props.handleElement}
+            onDelete={props.handleUnselectRow}
+          />
+        )}
+        info={(props) => {
+          const propsInfo = {
+            zone: props?.selectedRows?.[0]?.original,
+            selectedRows: props?.selectedRows,
+          }
+          props?.gotoPage && (propsInfo.gotoPage = props.gotoPage)
+          props?.unselect && (propsInfo.unselect = props.unselect)
+
+          return <InfoTabs {...propsInfo} />
+        }}
+      />
+    </TranslateProvider>
+  )
+}
+
+/**
+ * Displays details of a Zone.
+ *
+ * @param {Zone} zone - Zone to display
+ * @param {Function} [gotoPage] - Function to navigate to a page of a Zone
+ * @param {Function} [unselect] - Function to unselect
+ * @returns {ReactElement} Zone details
+ */
+const InfoTabs = memo(({ zone, gotoPage, unselect }) => {
+  const [get, { data: lazyData, isFetching }] = useLazyGetZoneQuery()
+  const id = zone?.ID ?? lazyData?.ID
+
+  const { settings: { FIREEDGE: fireedge = {} } = {} } = useAuth()
+  const { FULL_SCREEN_INFO } = fireedge
+  const { isFullMode } = useGeneral()
+  const { setFullMode } = useGeneralApi()
+
+  useEffect(() => {
+    !isFullMode && gotoPage()
+  }, [])
+
+  return (
+    <Stack overflow="auto">
+      <Stack
+        direction="row"
+        alignItems="center"
+        justifyContent="space-between"
+        gap={1}
+        mx={1}
+        mb={1}
+      >
+        <Stack direction="row">
+          {FULL_SCREEN_INFO === 'true' && (
+            <SubmitButton
+              data-cy="detail-back"
+              icon={<NavArrowLeft />}
+              tooltip={Tr(T.Back)}
+              isSubmitting={isFetching}
+              onClick={() => unselect()}
+            />
+          )}
+        </Stack>
+
+        <Stack direction="row" alignItems="center" gap={1} mx={1} mb={1}>
+          {FULL_SCREEN_INFO === 'false' && (
+            <SubmitButton
+              data-cy="detail-full-mode"
+              icon={isFullMode ? <Collapse /> : <Expand />}
+              tooltip={Tr(T.FullScreen)}
+              isSubmitting={isFetching}
+              onClick={() => {
+                setFullMode(!isFullMode)
+              }}
+            />
+          )}
+          <SubmitButton
+            data-cy="detail-refresh"
+            icon={<RefreshDouble />}
+            tooltip={Tr(T.Refresh)}
+            isSubmitting={isFetching}
+            onClick={() => get({ id })}
+          />
+          {typeof unselect === 'function' && (
+            <SubmitButton
+              data-cy="unselect"
+              icon={<Cancel />}
+              tooltip={Tr(T.Close)}
+              onClick={() => unselect()}
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using Function.prototype.bind and arrows functions as attributes will negatively impact performance in React. Each time the parent is rendered, the function will be re-created and trigger a render of the component causing excessive renders and more memory use.